marimbabot_speech/src/speech2text.py

```diff
@@ -21,17 +21,19 @@ def __init__(self):
 		self.recognize_freq = 2  # Hz
 		self.recognize_rate = rospy.Rate(self.recognize_freq)
 		self.no_speech_prob_filter = 0.5
-		rospy.logdebug(f"cwd:{os.getcwd()}")
+		#rospy.logdebug(f"cwd:{os.getcwd()}")
 		self.prompt = self.generate_prompt()
 
 	def generate_prompt(self):
 		commands = get_commands()
-		base_prompt = '''Marimbabot is a marimba playing robot arm. The possible commands include:		
-		'''
+		base_prompt = '''We do a demo of Marimbabot a marimba playing robot arm. Now some demonstrations. marimbabot read. marimbabot play.'''
+		"""
 		for command in commands:
```

Comment:
**This is a comment**

```

		"""
		for command in commands:
			base_prompt += ''.join(f'{command.strip()}, ')
		"""
```

It it not intended, right? @yunlong-wang-cn

Reply:
Actually, I have a new version with the PR #253, And this will be overwriten.